add-ons/istio/README.adoc

```diff
@@ -19,51 +19,31 @@ Considering this, it is ideal to setup a Minishift profile with the following co
 $ minishift profile set servicemesh <1>
 $ minishift config set memory 8GB <2>
 $ minishift config set cpus 3 <3>
-$ minishift config set openshift-version v3.7.0 <4>
-$ minishift addon enable admin-user <5>
-$ minishift addon enable anyuid <6>
-$ minishift start <7>
+$ minishift addon enable dynamic-admission-controllers <4>
```

Comment:
this could be made optional, as it not mandatory to have automatic sidecar injections to be enabled

Reply:
I agree and commented it.